package.json

```diff
@@ -109,9 +109,13 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^12.13.0 || >=14.15.0",
+    "node": "^12.20.2 || >=14.15.4",
```

Comment:
Why change the minor/patch versions? These are the LTS releases for v12 and v14.

Reply:
Was previously suggested by @rarkins , so i've updated v12 to mostly latest lts too. We can revert this.

Reply:
Sorry, I didn't see a previous comment on the `package.json`.

I think setting the minimum bar at the LTS version is what we normally do and raising it to versions released < 1 month ago is only going to cause annoyance for users.

Reply:
```suggestion
    "node": "^12.13.0 || >=14.15.0",
```
So reverting this

Reply:
What about the `engines-next.node: >=14.15.4` ?

Reply:
Yes, I think that should match. `engines-next.node: >=14.15.0`

Reply:
Done ✅